src/md/compiler/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(MDCOMPILER_SOURCES
   import.cpp
   importhelper.cpp
   mdutil.cpp
+  newmerger.cpp
```

Comment:
newmerger.cpp should be out. It is pretty sizable chunk and it is only used by C++ link.exe, nothing else is using it. It should be fine to delete it.